### PR TITLE
device: support unknown USB-connected and Bluetooth devices

### PR DESCRIPTION
**Pull request by pfps**
Support USB-connected and Bluetooth devices that do not have any entry in descriptors.py by matching udev devices that have Logitech for vendor and a product ID in the ranges that Logitech uses for these devices.

### CODE REVIEW

**Comment by pfps** [comment]
Please test if you have a device that connects via a USB cable or via Bluetooth.   Use `-d` when testing.  Test that Solaar starts up normally and behaves correctly when you unplug and re-plug or turn off and turn on your device(s).  Then comment out the record for your device(s) in lib/logitech-receiver/descriptors.py and test again.  The only change should be that the name of your device(s) at the top of the device pane might change and the short name in the receivers-and-devices pane is likely to change.

If any of your devices don't have an entry in descriptors.py please provide their name and a good short name for them.

@barolo Can you test this out?

File: docs/devices.md [hunk 1]
```diff
@@ -29,20 +29,19 @@ Unifying receiver, a nano receiver, or a lightspeed receiver. Add a line to
...
 the device and a feature list can speed up Solaar startup a bit. The
 arguments to the _D function are the device's long name, its short name
 (codename), its HID++ protocol version, its wireless product ID (wpid), and
 a tuple of known feature settings (from `lib/logitech/settings_templates.py`).
+If the device can connect via a USB cable its USB product ID should be included.
+If the device can connect via Bluetooth its Bluetooth product ID should be included.
 
-If a device connects via a USB cable or via Bluetooth its USB or Bluetooth product ID
-has to be added in `lib/logitech/descriptors.py`.
+Some Logiitech devices are not suited for use in Solaar.  If the device only
```
Comment (Klap-in) [inline docs/devices.md:41]: Logiitech -> Logitech

Comment (pfps) [reply docs/devices.md]: Fixed, thanks.

**Comment by barolo** [comment]
I'm curently unable to test that due to the fact that suddenly I can't pair my devices via BT at all....

**Pull request merged**